interface-definitions/interfaces-bridge.xml.in

```diff
@@ -126,6 +126,45 @@
                   </completionHelp>
                 </properties>
                 <children>
+                  <leafNode name="native-vlan">
+                    <properties>
+                      <help>Set the interface in ACCESS mode, automatically append vlan id to packets without the vlan id  in the incoming traffic, and automatically strip the vlan id when exporting packets with specific vlan id</help>
+                      <completionHelp>
+                        <list>disable</list>
+                      </completionHelp>
+                      <valueHelp>
+                        <format>1-4094</format>
+                        <description>Virtual Local Area Network (VLAN) ID</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>disable</format>
+                        <description>Disable the Virtual Local Area Network (VLAN) native to the interface</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="numeric" argument="--range 1-4094"/>
+                        <regex>disable</regex>
+                      </constraint>
+                      <constraintErrorMessage>VLAN ID must be between 1 and 4094</constraintErrorMessage>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="allowed-vlan">
+                    <properties>
+                      <help>Allow the bridge to pass the tagged VLAN tag on this interface</help>
+                      <valueHelp>
+                        <format>&lt;1-4096&gt;</format>
```

Comment:
Please make this more user readable like the following:
```
Possible completions:
   <n>     Individual Virtual Local Area Network (VLAN) ID
   <n-m>   Virtual Local Area Network (VLAN) ID Range
```

Reply:
Please also note that the VLAN range is only 1-4094 as 0 and 4095 are reserved and can not be used. Also your above validator is restricted to this range.

Reply:
In fact, I originally used this validator, but because of some problems, the validation logic is now moved to the configuration script